### PR TITLE
Document next survey wave and how to get involved

### PR DESCRIPTION
**Pull request by capnrefsmmat**
@Akvannortwick Looping you in -- let me know how you'd like to be involved here. This documentation is written for scientists who have access to our raw survey data, and it may be helpful if we're able to get you involved in future revisions and expansions to this documentation. But I don't know your current workload, so you can also tell me to go away.

### CODE REVIEW

**Comment by Akvannortwick** [comment]
I can jump in on this. If something changes I can let you know.

File: docs/symptom-survey/collaboration-revision.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Delphi continues to revise the COVID-19 Symptom Survey instruments in order to
+prioritize items that have the greatest utility for the response to the COVID-19
+pandemic. We conduct revisions in collaboration with data users, fellow
+researchers, and public health officials, to ensure the survey data best serves
+public health and research goals.
+
+## Survey Revisions
+
+If there is revision or question you would like us to consider please fill out
```
Comment (Akvannortwick) [inline docs/symptom-survey/collaboration-revision.md:17]: I think there is an article "a"  missing before revision.

File: docs/symptom-survey/coding.md [hunk 2]
```diff
@@ -268,3 +268,68 @@ your protocol.
...
+* Vietnamese
+
+**Draft** files:
+
+* [Survey text and coding](waves/Survey_of_COVID-Like_Illness_-_Wave_5.pdf)
+  (PDF)
+
+### Summary of Changes
+
+Wave 5 is a minor change to the survey instrument with the addition of a few new
```
Comment (Akvannortwick) [inline docs/symptom-survey/coding.md:292]: Would it be better said as:
"Wave 5 features a minor change..."

**Comment by capnrefsmmat** [comment]
@Akvannortwick  I've pushed a change to fix those two issues. Let me know if you have other comments or if this is ready to go

**Comment by Akvannortwick** [comment]
@capnrefsmmat Let me give it one more pass over and I will ping you when I am done. I am pretty sure those were the only big things that stuck out to me but never hurts to double-check.

**Pull request merged**